pkg/domain/infra/abi/play.go

```
@@ -359,6 +414,11 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 	containerConfig.Image = containerYAML.Image
 	containerConfig.ImageID = newImage.ID()
 	containerConfig.Name = containerYAML.Name
+
+	if podName != "" {
```

maybe fail here if podName is not specified?